native/src/hooks/useLoadCityContent.ts

```diff
@@ -1,49 +1,99 @@
-import { useCallback } from 'react'
-
-import { CityModel, Endpoint, ErrorCode, LanguageModel, ReturnType, useLoadAsync } from 'api-client'
+import moment from 'moment'
```

Comment:
This is sadly a little hard to review, let me know if you have ideas for improvements here. Basically the changes is just that we now always load all the offline available content here now at once here (languages, categories, events, pois) instead of heaving multiple hooks that do it separately. Only if everything succeeds, the whole loading of the hook succeeds. This is basically the behavior that was used before in our `loadCityContent` saga. This is also necessary at that point because we only store one last update time stamp.